iina/KeyCodeHelper.swift

```diff
@@ -294,40 +318,58 @@ class KeyCodeHelper {
     return keyString
   }
 
-  // Normalizes a single "press" of possibly multiple keys (as joined with '+')
+  /// Normalizes a single "press" of possibly multiple keys (as joined with '+')
+  /// _Examples:
+  /// Ex01: "Shift+-" → "_"
+  /// Ex02: "Shift+=" → "PLUS"
+  /// Ex03: "+-+" → "PLUS-PLUS"
+  /// Ex04: "meta+shift+k" → "Meta+K"
+  /// Ex05: "esc" → "ESC"
+  /// Ex06: "CTRL+SHIFT+SHIFT+SHIFT+x" → "Ctrl+X"
+  /// _
```

Comment:
In Xcode, clicking on `normalizeSingleMpvKeystroke` and then on the `?` button Xcode will show you what the rendered documentation looks like. 
```text
Normalizes a single “press” of possibly multiple keys (as joined with ‘+’) Examples: Ex01: “Shift+-” → “” Ex02: “Shift+=” → “PLUS” Ex03: “+-+” → “PLUS-PLUS” Ex04: “meta+shift+k” → “Meta+K” Ex05: “esc” → “ESC” Ex06: “CTRL+SHIFT+SHIFT+SHIFT+x” → “Ctrl+X” _
```

Markdown squishes it into one line. Maybe make use of a Markdown numbered list?

With a quick change to:
```swift
/// Normalizes a single "press" of possibly multiple keys (as joined with '+')
///
/// _Examples:_
/// 1. "Shift+-" → "_"
/// 2. "Shift+=" → "PLUS"
/// 3. "+-+" → "PLUS-PLUS"
/// 4. "meta+shift+k" → "Meta+K"
/// 5. "esc" → "ESC"
/// 6. "CTRL+SHIFT+SHIFT+SHIFT+x" → "Ctrl+X"
```

Xcode shows:

![issue-4267](https://user-images.githubusercontent.com/86170219/226032115-f624bfb1-e1b6-4b62-a3d5-e69d63807351.jpg)


Could still be improved. I have the [Basic Syntax](https://www.markdownguide.org/basic-syntax/) and [Extended Syntax](https://www.markdownguide.org/extended-syntax/)  pages from the [Markdown Guide](https://www.markdownguide.org/) site bookmarked.

There are various apps that support editing Markdown. I've been using the open source [Joplin](https://joplinapp.org/) application. Sources can be found on GitHub [here](https://github.com/laurent22/joplin).